pubspec.yaml

```diff
@@ -106,6 +107,9 @@ flutter:
   generate: true
 
   assets:
+    - path: assets/
```

Comment:
this actually increases app bundle size, cause it includes eveyrything in the assets folder, and we do not use everything there (e.g. banner). Maybe let's try to place all svgs in /assets/svgs ale set this path here. Let's see how this affects the size?